detox/test/e2e/03.actions.test.js

```diff
@@ -182,10 +182,15 @@ describe('Actions', () => {
     await expect(element(by.id('UniqueId007'))).toBeVisible();
   });
 
-  it(':ios: should adjust slider and assert its value', async () => {
+  it('should adjust slider and assert its value', async () => {
     await expect(element(by.id('sliderWithASimpleID'))).toHaveSliderPosition(0.25);
     await element(by.id('sliderWithASimpleID')).adjustSliderToPosition(0.75);
     await expect(element(by.id('sliderWithASimpleID'))).toHaveSliderPosition(0.75);
-    await expect(element(by.id('sliderWithASimpleID'))).toHaveValue("75%");
+    await expect(element(by.id('sliderWithASimpleID'))).toHaveSliderPosition(0.74, 0.1);
+
+    // on ios the accessibilityLabel is set to the slider value, but not on android
+    if (device.getPlatform() === 'ios') {
```

Comment:
@jonathanmos, do you think "%" handling can be taken to the JavaScript side here, or it won't be enough to cover the functionality?